capstone/files/deploy.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+NC='\033[0m'
+YELLOW='\033[1;33m'
+GREEN='\033[0;32m'
+
+set -e
+
+echo -e "${YELLOW}Build docker image...${NC}"
+docker build -t files-app .
+echo -e "${GREEN}Image built!${NC}"
+
+echo -e "${YELLOW}Save image...${NC}"
+docker save -o ./files-app-image files-app
+echo -e "${GREEN}Image saved!${NC}"
+
+echo -e "${YELLOW}Upload to server...${NC}"
+scp files-app-image wazuh-agent:~
```

Comment:
Add a readme or at least a comment specifying the necessary SSH configuration to run this.

Reply:
Done